aiohttp/helpers.py

```diff
@@ -514,3 +515,22 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     def _cancel_task(self):
         self._cancelled = self._task.cancel()
+
+
+def stop_listening(srv, loop):
```

Comment:
This looks weird. I thought is enough to call `.close()` method of `protocol` instance.